public_html/wp-content/mu-plugins/latest-site-hints.php

```diff
@@ -2,11 +2,17 @@
 
 namespace WordCamp\Latest_Site_Hints;
 use function WordCamp\Sunrise\get_top_level_domain;
-
 use const WordCamp\Sunrise\{ PATTERN_YEAR_DOT_CITY_DOMAIN_PATH, PATTERN_CITY_SLASH_YEAR_DOMAIN_PATH };
 
 defined( 'WPINC' ) || die();
 
+if ( 2 === SITE_ID_CURRENT_SITE ) {
```

Comment:
Probably better to introduce a constant for the Network IDs, to avoid multiple different locations of IDs and switching methods (here it's network ID, elsewhere it's domain)
```suggestion
if ( NETWORK_ID_EVENTS === SITE_ID_CURRENT_SITE ) {
```

Reply:
That makes sense 👍🏻